x/evm/keeper/grpc_query.go

```diff
@@ -276,6 +276,34 @@ func (q Querier) Event(c context.Context, req *types.EventRequest) (*types.Event
 	return &types.EventResponse{Event: &event}, nil
 }
 
+// DepositState fetches the state of a deposit confirmation using a grpc query
```

Comment:
that description seems odd, why do you mention grpc query here?

Reply:
lol it is a copy/paste of what was there before. I'll re-write this.